src/surfaces/filtersRail/filtersRail.jsx

```diff
@@ -290,7 +290,7 @@ function FiltersRail(props) {
                             <div className={actionButtonsClasses}>
                                 <A
                                     className={clearFiltersClasses}
-                                    disable={!filterOptions.isDirty || !filterOptions.isFiltering}
+                                    disable={!filterOptions.isDirty && !filterOptions.isFiltering}
```

Comment:
I knew this was a bit tricky and hard to follow! =)
What if we got rid of the NOTs and made it an OR -- would that be easier to read/understand?
```JavaScript
disable={filterOptions.isDirty || filterOptions.isFiltering}
```

Reply:
Yeah, you previously commented on one of the PR about that same condition. Stupid of me, should have double checked it 😞 It's confusing as well =)